r124/u_inref/ydb312_gtm8182g.csh

```diff
@@ -0,0 +1,95 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+setenv gtm_repl_instance "mumps.repl"
+unsetenv gtm_db_counter_sem_incr # avoid semaphore counter overflow
+
+#setenv ydb_gbldir "GBL3"
```

Comment:
Please remove commented lines if they are no longer relevant. They are found in various sections of this .csh file.